src/librustc_session/options.rs

```diff
@@ -939,6 +939,8 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         "support compiling tests with panic=abort (default: no)"),
     parse_only: bool = (false, parse_bool, [UNTRACKED],
         "parse only; do not compile, assemble, or link (default: no)"),
+    partitioning_strategy: Option<String> = (None, parse_opt_string, [TRACKED],
```

Comment:
Maybe this should be `-Zcgu-partitioning-strategy` instead?

Reply:
I think `cgu-partitioning-strategy` would be best.

Reply:
I agree (and went with `cgu-partitioning-strategy`)